packages/flutter/test/material/about_test.dart

```diff
@@ -742,6 +742,158 @@ void main() {
     expect(nestedObserver.licensePageCount, 0);
   });
 
+  group('Barrier dismissible', () {
+    testWidgets('Barrier is dismissible with default parameter', (WidgetTester tester) async {
+      final AboutDialogObserver rootObserver = AboutDialogObserver();
```

Comment:
Ah I guess this rootObserver needed to be recreated each time?  Could you use setUp to create it before each test instead of setUpAll?  This way is fine too, though.